Tasks/Maven/maven.ps1

```diff
@@ -79,13 +85,18 @@ Invoke-Maven -MavenPomFile $mavenPOMFile -Options $options -Goals $goals
 PublishTestResults $publishJUnitResults $testResultsFiles
 
 # Publish code coverage
-PublishCodeCoverage  $isCoverageEnabled $mavenPOMFile $CCReportTask $summaryFile $reportDirectory $codeCoverageTool 
+PublishCodeCoverage  $isCoverageEnabled $mavenPOMFile $CCReportTask $summaryFile $reportDirectory $codeCoverageTool $reportPOMFile
```

Comment:
Why publishCodeCoveage commandlet requires POM.xml. How will this work in publish code Coverage task